### ISSUE TITLE
Web component: Filter Panel - 'Clear' option is appearing at the top of Filter.

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#6667]
### The problem
Issue:
By default clear option is not available then after selecting filter
Clear option is appearing at the top of the filter, which makes the screen reader user to come back to top to clear it. 
Sometimes users wont know about this clear option available until they come back to top and verify.

**Comment by jeffchew** [comment]
This is a design/accessibility question. Placement of the clear button would be more ideal possibly at the bottom or to show a disabled button at the top, which becomes enabled once things are selected.

**Comment by RichKummer** [comment]
This is for the tablet and mobile breakpoints where screen reader users would not necessarily know there is a clear function until they get to the bottom of the filter panel. Ideal in all screen sizes is to tab through the disabled Clear button (this way, users knows this is available) before getting to filter options.

**Comment by RobertaJHahn** [comment]
@jeffchew It looks like this issue is in Pending Scheduling, but it is a Feature Request without a sizing.  I have added it to the July 26 issues refinement agenda and moved back to the "To be Traiged" pipeline.

**Comment by RobertaJHahn** [comment]
@shixiedesign Could you review this with the designers?  Thanks.

**Comment by shixiedesign** [comment]
@Praveen-sr 

Design team reviewed together and decided this is not a feature we should add at this time. Reasons being: 
- if we move the "Clear" button to the bottom of the panel, keyboard users still have to move through a lot of key strokes, and users with 400% zoom will still not see it appearing
- we cannot move "Clear" to each individual accordions because it would interfere with Carbon accordion, which makes the whole bar area clickable. To add a button there we'd have to create a new accordion component
- Upon filtering, we expect the focus to move to content. So if filters need to be adjusted, it's a back-tracking logic to look for filter panel already, so hopefully it won't be hard to find the "Clear" button. 

That said, I think we should all keep our eyes and ears open for alternative solutions. Please let us know if anything  we didn't think of.

**Comment by RobertaJHahn** [comment]
Moving to Icebox, based on Shixie's comments above.

**Comment by shixiedesign** [comment]
Added above design decision / logic into Filter panel design spec. Closing. ![image.png]
(https://images.zenhubusercontent.com/5be09c3a6102900c6d16d991/fe920f53-9464-4b32-9891-3cb2fd21afef)
